src/shogun/machine/LinearMachine.cpp

```diff
@@ -120,3 +141,50 @@ void CLinearMachine::store_model_features()
 {
 }
 
+float64_t CLinearMachine::compute_bias(CFeatures* data)
+{
+	REQUIRE(m_labels,"No labels set\n");
+
+	if (!data)
+		data=features;
+
+	REQUIRE(data,"No features set\n");
+
+	REQUIRE(m_labels->get_num_labels() == data->get_num_vectors(),"Number of training vectors does not match number of labels\n");
```

Comment:
please respect the 80 char margin
Also, please writter better error message, see the development guidelines in the wiki.
The point is to say "Number of training vectors (%d) does not match number of labels (%d)" so that users get a hint on where they made a mistake